includes/admin.form.inc

```diff
@@ -22,6 +22,12 @@ function islandora_batch_admin_form($form, $form_state) {
     '#description' => t('When a batch finishes, we will attempt to render a link to review the queue for the given set. This is dependent on the "views" module being enabled.'),
     '#default_value' => variable_get('islandora_batch_link_when_finished', TRUE),
   );
+  $form['islandora_batch_auto_remove_set'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Auto-remove batch set when processing finishes?'),
+    '#description' => t('When a batch set is processed, it will be removed from storage.'),
```

Comment:
Seems like this should mention something about "success" being another criteria, and "storage" could be misunderstood. Something like: "When a batch set successfully completes, it will be cleared from the batch queue.", Maybe?

... I'm not great at documentation-type things...